agriculture_pricing/README.md

```diff
@@ -1,3 +1,154 @@
 # Agriculture Pricing Example
 
-Example of implementation of agriculture pricing defined [here](https://github.com/Gurobi/modeling-examples/tree/master/agricultural_pricing).
\ No newline at end of file
+Example of implementation of agriculture pricing adapted from [here](https://github.com/Gurobi/modeling-examples/tree/master/agricultural_pricing).
+
+## Data View
+
+Columns marked with __*__ are index columns.
+
+#### `capacity.csv`
+
+| `component`* | `capacity` |
+| --- | --- |
+| __Defines__ set <u>Components</u> ($C$) | __Defines__ param <U>Last Year's Demand |
+
+> Missing index values allowed? : N/A
+
+#### `composition.csv`
+
+| `dairy`* | `component`* | `composition` |
+| --- | --- | -- |
+| Elements __belong__ to <u>Dairy ($D$)</u> | Elements __belong__ to set <u>Components ($C$)</u> | __Defines__ param <u>Composition</u> |
+
+> Missing index values allowed? : No
+
+#### `constants.csv`
+
+| `price_index` |
+| --- |
+| __Defines__ param <u>Price Index</u> |
+
+> Missing index values allowed? : N/A
+
+#### elasticity.csv
+
+| `dairy_1`* | `dairy_2`* | `elasticity` |
+| --- | --- | -- |
+| Elements __belong__ to <u>Dairy ($D$)</u> | Elements __belong__ to <u>Dairy ($D$)</u> | __Defines__ param <u>Elasticity</u> |
+
+> Missing index values allowed? : Yes
+
+#### `market.csv`
+
+| `dairy`* | `demand` | `price` |
+| --- | --- | -- |
+| __Defines__ set <u>Dairy</u> ($D$) | __Defines__ param <u>Last Year's Demand | __Defines__ param <u>Last year's price |
+
+> Missing index values allowed? : N/A
+
+## Model View
+
+
+#### Dairy (Set), $\text{D}$
+
+> Source: From `index columns` of `market.csv`.
+
+#### Components (Set), $\text{C}$
+
+> Source: From `index columns` in `capacity.csv`.
+
+#### Capacity (Param), $\text{capacity}_c$
+
+$$0 \le \text{capacity}_{c} \quad \forall c \in C$$
+
+> Description: Yearly availability of component $c$ (1000 tons).
+
+> Source: From column `capacity` in `capacity.csv`.
+
+#### Composition (Param), $\text{qtyper}_{c,d}$
+
+$$0 \le \text{qtyper}_{c,d}\le 1 \quad \forall c\in C, d\in D$$
+
+> Source: From column `composition` in `composition.csv`
+
+> [!WARNING]
+> How do we specify which column is which in `composition.csv`
```

Comment:
Let me know if I misunderstood your questions, but...
we have an SQL table called component_to_sets(component_id, set_id, order). The order allows us to specify which set corresponds to which index. https://www.notion.so/SQL-Agricultural-Pricing-martin-s-method-89e347bb1c0b4fdba27943ea17d64059
I have the order in the hundreds because we visualized this being a drag-and-drop to arrange the order.

Reply:
(I was more wondering UI wise how we might do this)